migration.md

```diff
@@ -0,0 +1,30 @@
+### `@uppy/informer`
```

Comment:
why are there two migration files? shouldn't this be put inside .github/MIGRATION.md too?

Reply:
removed  in #5825